nemo/collections/asr/models/configs/msdd_config.py

```diff
@@ -0,0 +1,198 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from dataclasses import asdict, dataclass
+from typing import Dict, Optional, Tuple, Union
+
+import torch
+from lightning_fabric.utilities.exceptions import MisconfigurationException
+
+DEFAULT_DOMAIN_PARAMETERS = {
```

Comment:
Let's use data class please instead of nested dicts

Reply:
@SeanNaren Since we already made default parameters with classes, can we just use yaml file from the git?

Reply:
Removed the need for this as we do not have domain anymore!